app/views/news/2018-08-08-july-2018-monthly-update.html.md

```diff
@@ -0,0 +1,71 @@
+---
+summary: In July, we paid for 90 hours of developer work, released Bundler 1.16.3, and shipped the alpha release of two-factor authentication for RubyGems.org.
+---
+
+Hello! Welcome to the monthly update. During July, our work was supported by [Coinbase](https://coinbase.com), [Cloud City Development](http://www.cloudcity.io), [reinteractive](https://reinteractive.com/), and many others.
+
+## ruby together news
+
+In July, Ruby Together was supported by 74 different companies, including Sapphire member [Stripe](https://stripe.com). In total, we were supported by 79 individual members and 66 friends of Ruby Together. Thanks to all of our members for making everything that we do possible!
+
+The Ruby Together Board of Directors met to finalize and ratify a plan for a new Ruby open source mentorship program called RubyMe—stay tuned for future updates about the program!
+
+In community news, Developer Evangelist [PJ Hagerty](https://twitter.com/aspleenic) continued to spread the word about Ruby Together at RubyConf Kenya. Our Google Summer of Code (GSoC) students made progress on the `bundle change` feature, and they also worked on a comment that rewrites a Gemfile into a canonical form. And [Kerri Miller](https://twitter.com/kerrizor) addressed outstanding issues in the RubyGems and RubyGems.org support queues. 
+
+## bundler news
+
+Last month, we released Bundler 1.16.3. The new release includes improvements such as [FEATURES], and fixes several bugs including [BUGS]. Since we announced the new RFC process last month, we’ve seen a few request for comments come into the [Bundler RFC repo](https://github.com/bundler/rfcs/pulls).
+
+In other exciting news, we added a new contributor to the core team. Welcome [Deivid Rodriguez](https://github.com/deivid-rodriguez) to Bundler! :tada:
```

Comment:
>In other exciting news, we added a new contributor to the core team. Welcome [Deivid Rodriguez](https://github.com/deivid-rodriguez) to Bundler! 

@indirect we should probably list out some examples of Deivid's work on Bundler here, too!